alf/drivers/policy_driver.py

```diff
@@ -173,12 +173,14 @@ def predict(self, max_num_steps, time_step, policy_state):
             name="predict_loop")
         return time_step, policy_state
 
-    def _summary(self, loss_info, grads_and_vars):
+    def _summary(self, training_info, loss_info, grads_and_vars):
         if self._summarize_grads_and_vars:
             eager_utils.add_variables_summaries(grads_and_vars,
                                                 self._train_step_counter)
             eager_utils.add_gradients_summaries(grads_and_vars,
                                                 self._train_step_counter)
+            common.add_action_summaries(training_info.action,
```

Comment:
I didn't notice that  common.add_action_summaries() was removed in off_policy_driver PR.